configs/openSUSE/dbus-services.toml

```diff
@@ -33,10 +30,14 @@ digests = [
 [[FileDigestGroup]]
 package = "cups"
 type = "dbus"
-note = ""
-bug = "bsc#515977"
-nodigests = [
-    "/etc/dbus-1/system.d/cups.conf",
+note = "the well known printing system daemon"
+bugs = [ "bsc#515977", "bsc#1201346" ]
+digests = [
+    {
+        path = "/etc/dbus-1/system.d/cups.conf",
```

Comment:
I thought bsc#1201346 was about moving the config file from `/etc/dbus-1/system.d/cups.conf` to `/usr/share/dbus-1/system.d/cups.conf`. Is this path right?

Reply:
Good point. Seems I was too busy with the digests to do the actual task ;). I addressed this now.